JetMETCorrections/Type1MET/plugins/PFCandMETcorrInputProducer.cc

```diff
@@ -12,29 +12,25 @@ PFCandMETcorrInputProducer::PFCandMETcorrInputProducer(const edm::ParameterSet&
     vParameterSet cfgBinning = cfg.getParameter<vParameterSet>("binning");
     for ( vParameterSet::const_iterator cfgBinningEntry = cfgBinning.begin();
 	  cfgBinningEntry != cfgBinning.end(); ++cfgBinningEntry ) {
-      binning_.push_back(new binningEntryType(*cfgBinningEntry));
+      binning_.emplace_back(new binningEntryType(*cfgBinningEntry));
     }
   } else {
-    binning_.push_back(new binningEntryType());
+    binning_.emplace_back(new binningEntryType());
   }
   
-  for ( std::vector<binningEntryType*>::const_iterator binningEntry = binning_.begin();
+  for ( auto binningEntry = binning_.begin();
```

Comment:
Other people (especially those that wrote this code in the past) will have to read it again, seems like a bad idea to confuse them.